.github/workflows/docker-beta.yaml

```
@@ -0,0 +1,33 @@
+
+# Builds the hardened two-stage Dockerfile.twostage and publishes it under the
+# :beta tag for staging validation. The production Dockerfile and its publish
+# pipeline (docker.yaml) are intentionally left untouched, so this cannot affect
+# the customer-facing :latest image.
+
+jobs:
+  beta-build-and-push:
+    name: Build Dockerfile.twostage and push :beta
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)